src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFiles.java

```diff
@@ -186,11 +186,11 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
             return;
         }
 
-
-        // create VET output
-        if (!variant.isReferenceBlock()) {
+        // write to VET if NOT reference block and NOT a no call (set GQ to 0 for variant)
```

Comment:
the "set GQ to 0 for variant" applies later to the PET table section, not to this section